cypress/e2e/index.cy.js

```diff
@@ -97,4 +108,66 @@ describe('percySnapshot', () => {
       });
     });
   });
+
+  describe('withRetry Function Test', () => {
```

Comment:
while you have added a spec for this function, there is no test confirming that this retry code is used in main function